### PR TITLE
Remove alpha from kubectl debug command invocation

### PR DESCRIPTION
**Pull request by verb**
**What type of PR is this?**

/kind feature
/sig cli
/priority important-soon

**What this PR does / why we need it**: This PR graduates `kubectl alpha debug` to `kubectl debug`

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
`kubectl alpha debug` has graduated to beta and is now `kubectl debug`.
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs
- [KEP]: https://github.com/kubernetes/enhancements/tree/master/keps/sig-cli/1441-kubectl-debug
```

### CODE REVIEW

**Comment by verb** [comment]
This should not merge until after #96058 and [graduation criteria](https://github.com/kubernetes/enhancements/tree/master/keps/sig-cli/1441-kubectl-debug#alpha---beta-graduation) are met.

/hold

File: staging/src/k8s.io/kubectl/pkg/cmd/alpha.go [hunk 1]
```diff
@@ -36,7 +35,7 @@ func NewCmdAlpha(f cmdutil.Factory, streams genericclioptions.IOStreams) *cobra.
 
 	// Alpha commands should be added here. As features graduate from alpha they should move
 	// from here to the CommandGroups defined by NewKubeletCommand() in cmd.go.
-	cmd.AddCommand(debug.NewCmdDebug(f, streams))
+	//cmd.AddCommand(example.NewCmdExample(f, streams))
```
Comment (soltysh) [inline staging/src/k8s.io/kubectl/pkg/cmd/alpha.go:38]: I wonder if we can make this work for both places at least for one release. I know it's not a hard requirement, but I think it's good UX to have both working. The alpha one can have a deprecation warning.

**Review by soltysh — APPROVED** [review]
/approve
one nit and this is good to go

**Comment by verb** [comment]
we've met the merge criteria

/hold cancel

**Review by soltysh — APPROVED** [review]
/lgtm
/triage accepted

**Pull request merged**